main.go

```diff
@@ -2,27 +2,29 @@ package main
 
 import (
 	"crypto/rand"
+	"flag"
 	"fmt"
-	"io"
-
 	"github.com/shirou/gopsutil/mem"
+	"io"
 )
 
 var (
-	times     = 3
+	//times     = 3
 	writeSize = uint64(1)
 )
 
 func main() {
 	totalMem, _ := mem.VirtualMemory()
 	maxMemWrite := totalMem.Available / 2
-	for times > 0 {
+	setTimes := flag.Int("n", 1, "integer, please")
```

Comment:
You'll find it a common pattern to handle configuration and flag parsing outside of the main business logic.
How might you get your integer set outside of the `main`? See if you can include the resolution of the flag pointer.